frontend/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    "react",
+    "es2015"
+  ],
+  "plugins": [
+    "transform-class-properties"
+  ]
+}
```

Comment:
This file just sets up babeljs to handle react and es6 code. I was hoping to have no babel plugins (and just pure es6), but react code is much more natural to write with transform-class-properties so I bit my tongue.

without transform-class-properties, the component below would need to have a constructor and call `.bind(this)` on every function (including render) to be able to access the react props... it's just dumb. It's much simpler to use arrow functions for every function in the react component, and this works exactly how you want it to.